mw-webapp/src/globalStore/ThemeStore.ts

```diff
@@ -53,6 +53,10 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.DARK]: "#8F8F8F",
     [Theme.LIGHT]: "#A8A1DD",
   },
+  secondaryCardsBgColor: {
```

Comment:
do we have primaryCradsBgColor? If not then name is not actual. You can use name for example additionalInfoCardsBgColor

Reply:
No, we don't have the primary. I renamed to  `additionalInfoCardBgColor`